tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
my point is the following - users can add new client to the requirements and use it in Scrapy Cloud. when they do so, they would probably use auth from the environment - and that's where *have* to pass decoded `job_key:JWT_token` pair, they don't have an option to pass an apikey and if they pass that pair as an apikey - their requests to dash endpoint will fail with 401 because Connection doesn't expect such input and uses apikey parameter as a "user" part of auth tuple